python/orca/example/learn/mmcv/faster_rcnn/README.md

```diff
@@ -0,0 +1,170 @@
+[>>>>>>>>>>>>>>                  ] 11/25, 0.7 task/s, elapsed: 16s, ETA:    20s
+[>>>>>>>>>>>>>>>                 ] 12/25, 0.7 task/s, elapsed: 17s, ETA:    18s
+[>>>>>>>>>>>>>>>>                ] 13/25, 0.7 task/s, elapsed: 18s, ETA:    17s
+[>>>>>>>>>>>>>>>>>               ] 14/25, 0.7 task/s, elapsed: 19s, ETA:    15s
+[>>>>>>>>>>>>>>>>>>>             ] 15/25, 0.7 task/s, elapsed: 20s, ETA:    14s
+[>>>>>>>>>>>>>>>>>>>>            ] 16/25, 0.7 task/s, elapsed: 22s, ETA:    12s
+[>>>>>>>>>>>>>>>>>>>>>           ] 17/25, 0.7 task/s, elapsed: 23s, ETA:    11s
+[>>>>>>>>>>>>>>>>>>>>>>>         ] 18/25, 0.8 task/s, elapsed: 24s, ETA:     9s
+[>>>>>>>>>>>>>>>>>>>>>>>>        ] 19/25, 0.8 task/s, elapsed: 25s, ETA:     8s
+[>>>>>>>>>>>>>>>>>>>>>>>>>       ] 20/25, 0.8 task/s, elapsed: 26s, ETA:     7s
+[>>>>>>>>>>>>>>>>>>>>>>>>>>      ] 21/25, 0.8 task/s, elapsed: 28s, ETA:     5s
+[>>>>>>>>>>>>>>>>>>>>>>>>>>>>    ] 22/25, 0.8 task/s, elapsed: 29s, ETA:     4s
+[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>   ] 23/25, 0.8 task/s, elapsed: 30s, ETA:     3s
+[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>  ] 24/25, 0.8 task/s, elapsed: 32s, ETA:     1s
+[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 25/25, 0.8 task/s, elapsed: 33s, ETA:     0s
+(MMCVRayEpochRunner pid=188889)
+(MMCVRayEpochRunner pid=188889)
+(MMCVRayEpochRunner pid=188889)
+(MMCVRayEpochRunner pid=188889) ---------------iou_thr: 0.5---------------
+(MMCVRayEpochRunner pid=188889) 2023-03-13 10:48:45,974 - mmdet - INFO -
+(MMCVRayEpochRunner pid=188889) +------------+-----+------+--------+-------+
+(MMCVRayEpochRunner pid=188889) | class      | gts | dets | recall | ap    |
+(MMCVRayEpochRunner pid=188889) +------------+-----+------+--------+-------+
+(MMCVRayEpochRunner pid=188889) | Car        | 62  | 632  | 0.984  | 0.791 |
+(MMCVRayEpochRunner pid=188889) | Pedestrian | 13  | 78   | 0.846  | 0.800 |
+(MMCVRayEpochRunner pid=188889) | Cyclist    | 7   | 66   | 0.286  | 0.052 |
+(MMCVRayEpochRunner pid=188889) +------------+-----+------+--------+-------+
+(MMCVRayEpochRunner pid=188889) | mAP        |     |      |        | 0.548 |
+(MMCVRayEpochRunner pid=188889) +------------+-----+------+--------+-------+
+(MMCVRayEpochRunner pid=188889) 2023-03-13 10:48:45,976 - mmdet - INFO - Epoch [1][25/25]       lr: 2.500e-03, eta: -1 day, 23:59:54, time: 5.530, data_time: 0.092, loss_rpn_cls: 0.0237, loss_rpn_bbox: 0.0160, loss_cls: 0.3650, acc: 87.1133, loss_bbox: 0.3418, loss: 0.7465, AP50: 0.5480, mAP: 0.5476
+Stopping orca context
+```
+
+- yarn-client mode
+
+If you want to use the pre-trained checkpoints, please first upload the checkpoint file to HDFS, then execute
+
+```bash
+python train.py  --dataset /your/dataset/path/to/kitti_tiny/ --cluster_mode yarn-client --config $MMDET_PATH/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py --load_from hdfs://ip:port/your/hdfs/path/to/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco_20210526_095054-1f77628b.pth
```

Comment:
Do we need to upload dataset to hdfs or pre-download it on each node?

Reply:
yes, I will add more explanation to clarify this.